src/Public/Logging.ps1

```diff
@@ -142,7 +179,51 @@ function New-PodeLoggingMethod {
 
         [Parameter(ParameterSetName = 'Custom')]
         [object[]]
-        $ArgumentList
+        $ArgumentList,
+
+        [Parameter(Mandatory = $true, ParameterSetName = 'Syslog')]
+        [switch]
+        $Syslog,
+
+        [Parameter(Mandatory = $true, ParameterSetName = 'Syslog')]
+        [string]
+        $Server,
+
+        [Parameter( ParameterSetName = 'Syslog')]
+        [Int16]
+        $Port = 514,
+
+        [Parameter( ParameterSetName = 'Syslog')]
+        [ValidateSet('UDP', 'TCP', 'TCPwithTLS', 'Splunk', 'LogInsight')]
+        $Transport = 'UDP',
+
+        [Parameter( ParameterSetName = 'Syslog')]
+        [System.Security.Authentication.SslProtocols]
+        $TlsProtocol = [System.Security.Authentication.SslProtocols]::Tls12,
+
+        [Parameter( ParameterSetName = 'Syslog')]
+        [switch]
+        $RFC3164,
```

Comment:
This might be better as a `SyslogFormat` param maybe, with a ValidateSet of RFC3184 and RFC5424 - with a default of RFC5424. Mostly in case a newer format is released